Platform/OpenCore/OpenCorePlatform.c

```diff
@@ -34,6 +34,7 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 #include <Guid/AppleVariable.h>
 
 STATIC CHAR8    mCurrentSmbiosProductName[48];
+STATIC UINT32   mPlatformFeature = 0;
```

Comment:
Please make this local. Global variable for SMBIOS is only needed because of the usage outside of the function.